.github/CODEOWNERS

```diff
@@ -0,0 +1,23 @@
+# Copied files:
+
+.github/dco.yml @IBM/cloudant-sdks
+scripts/setup_couch.sh @IBM/cloudant-sdks
+scripts/setup_wiremock.sh @IBM/cloudant-sdks
+stubs/mappings.json @IBM/cloudant-sdks
+KNOWN_ISSUES.md @IBM/cloudant-sdks
+Jenkinsfile @IBM/cloudant-sdks
+
+# Generated code files:
+
+src/main/java/com/ibm/cloud/cloudant/v1/* @IBM/cloudant-sdks
```

Comment:
I believe you need `**`?
[Example with `**`](https://github.com/dotnet/samples/blob/master/.github/CODEOWNERS#L18)

Reply:
Oh, I clicked on the big green button by mistake before I fixed these. I need to create another PR with the fixes.